### PR TITLE
Fixed offline save and load; did basic browse problems

### PR DESCRIPTION
**Pull request by tnguyen10**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,17 +16,14 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name="Activities.Login"
+            android:name="Activities.PatientHomeMenuActivity"
```
Comment (modelarious) [inline app/src/main/AndroidManifest.xml]: Could we leave Login as the main activity in the manifest?

**Comment by modelarious** [comment]
Apart from the one comment I had, everything looks good.  I'll push this once that's resolved

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (tnguyen10) [reply app/src/main/AndroidManifest.xml]: yes, sorry I was testing the activity to see if they work

**Comment by tnguyen10** [comment]
I've also added add problem activity

**Comment by tnguyen10** [comment]
I'll start working on the controller testing after this.

**Pull request merged**